### PR TITLE
Created blueprint for SRSRAN with onramp

### PR DESCRIPTION
**Pull request by sureshmarikkannu**
Once the PR https://github.com/opennetworkinglab/aether-srsran/pull/1 merged need to update the deps folder in this repo. We will do this as part of separate PR.

### CODE REVIEW

File: vars/README.md [hunk 1]
```diff
@@ -48,3 +48,6 @@ blueprints include:
   emulated or physical UE. Details documented
   [here](https://docs.aetherproject.org/master/onramp/blueprints.html#oai-5g-ran). 
 
+* `main-srsran.yml`: Configures the Core with an SRSRAN-based 5G RAN,
```
Comment (gab-arrobo) [inline vars/README.md:51]:
```suggestion
* `main-srsran.yml`: Configures the SD-Core with a srsRAN-based 5G RAN,
```

File: vars/main-srsran.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      name: host
+      subnet: "172.20.0.0/16"
+      bridge:
+        name: rfsim5g-public
+  simulation: true
+  gnb:
+    conf_file: deps/srsran/roles/gNb/templates/gnb_zmq.conf
+    ip: "172.20.0.2"
+  ue:
+    conf_file: deps/srsran/roles/uEsimulator/templates/ue_zmq.conf
```
Comment (gab-arrobo) [inline vars/main-srsran.yml:54]: Are you using all parameters/variables set here?

File: vars/README.md [hunk 3]
```diff
@@ -48,3 +48,6 @@ blueprints include:
...
+  supporting another gNB option. Details documented
+  TODO:.
```
Comment (gab-arrobo) [inline vars/README.md:53]: Are you planning to open a PR in the aether-docs?

File: vars/main-srsran.yml (hunk 2 above)
Comment (sureshmarikkannu) [reply vars/main-srsran.yml:54]: Yes, except data_iface/bridge.name all other parameters are used now. I am working on using separate n/w for deploying  srsran (like oai). For that purpose i kept these two parameters above.

File: vars/README.md (hunk 3 above)
Comment (sureshmarikkannu) [reply vars/README.md:53]: yes, i need to update the blueprint for srsRAN.

**Comment by gab-arrobo** [comment]
@sureshmarikkannu, you need to sign your commits by appending the `-s` flag. Otherwise, the DCO check fails

File: vars/main-srsran.yml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        core:   "192.168.250.3/24"
+      ue_ip_pool: "172.250.0.0/16"
+  amf:
+    ip: "10.76.28.113"
+
+srsran:
+  docker:
+    container:
+      gnb_image: aetherproject/srsran-gnb:0.0.1-dev
+      ue_image: aetherproject/srsran-ue:0.0.1-dev
```
Comment (gab-arrobo) [inline vars/main-srsran.yml:42]: Now we have "official" release images
```suggestion
      gnb_image: aetherproject/srsran-gnb:rel-0.0.1
      ue_image: aetherproject/srsran-ue:rel-0.0.1
```

**Review by gab-arrobo — CHANGES_REQUESTED** [review]

File: vars/main-srsran.yml (hunk 4 above)
Comment (sureshmarikkannu) [reply vars/main-srsran.yml]: done

**Comment by sureshmarikkannu** [comment]
> @sureshmarikkannu, you need to sign your commits by appending the `-s` flag. Otherwise, the DCO check fails

When i commit from GUI (Commit suggestion the flag was missing). I fixed it now.

**Comment by llpeterson** [comment]
Isolated enough to approve.

File: vars/main-srsran.yml [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    network:
+      data_iface: ens18
+      name: host
+      subnet: "172.20.0.0/16"
+      bridge:
+        name: rfsim5g-public
+  simulation: true
+  gnb:
+    conf_file: deps/srsran/roles/gNB/templates/gnb_zmq.conf
+    ip: "172.20.0.2"
```
Comment (mbilal92) [inline vars/main-srsran.yml:52]: Why do you need to fix this?

Comment (mbilal92) [reply vars/main-srsran.yml:52]: for your reference https://github.com/opennetworkinglab/aether-gnbsim/blob/master/roles/simulator/tasks/start.yml#L26

Comment (sureshmarikkannu) [reply vars/main-srsran.yml:52]: I just used the template from oai blueprint. Also currently the srsgnb/srsue deployed using host network and am working on running it on separate network. Once its fixed then i can use the mechanism you mentioned to retrieve gnb IP. Also i do see that while creating the container we are using this parameter to mention the IP address used for gnb container (in OAI blueprint). So we need to think about it (let me play around this and update).

**Pull request merged**